docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,37 @@
+# Troubleshooting
+
+If you are having problems with your AmpliPi device, start here.
+
+## Audio
+
+### Noise
+Audio noise can have a lot of causes, below are some common reasons:
+* **Mixing grounds**: AmpliPi has two different grounded lines, analog and digital. The most common way to mix them is by plugging in an audio device and attempting to power it by the usb on the rear of the unit, all usb-powered devices must be powered from a wall outlet
+* **Speaker setup**: Some speaker configurations can lead to excess noise being inducted into the output. This is unrelated to the system and has everything to do with how the speakers are wired and installed.
+* **Hardware issues**: See the support page for more info.
+
+### No Audio
+Lack of audio typically has one of a few causes:
+* **No Source / Bad Source**: The source itself isn't outputting audio properly, try changing to a different source to confirm
+* **Bad Output / Bad Zone**: Either the speaker, the speaker wires, or one of the ports on the back of the AmpliPro might be broken or hooked up incorrectly. Try a different speaker, length of wire, or port on the back of the AmpliPro to test each case
```

Comment:
I don't know if this section strictly belongs, and any info in it probably needs to be edited for tone just to keep it in line with the rest of the manual

I just added it because there was only one subheader under the audio header of the troubleshooting section and it felt like it needed another to not be clunky and awkward